simc.py

```diff
@@ -228,9 +231,14 @@ def check(addon_data):
                                     return
 
                     if data != 'addon':
-                        if check_spec(realm, char, api_key) == 'HEALING':
+                        api = await check_spec(region, realm, char, apikey)
```

Comment:
Possible you can change this line to `await check_spec(realm, char, api_key)` to match the main branch in var names and static region?